source/07-processing/04-counting-unique-titles.rst

```diff
@@ -7,9 +7,11 @@
 Counting Unique Titles
 ----------------------
 
-Some COUNTER Metric_Types count the number of unique titles that had a certain activity, such as a Unique_Title_Requests or Unique_Title_Investigations.
+Two COUNTER Metric_Types count the number of unique titles that had a certain activity: Unique_Title_Requests and Unique_Title_Investigations. These metrics apply only to books, including Data_Types Book and Reference_Work.
 
-For the purpose of COUNTER metrics, a title represents the parent work that the item is part of. When the item is a chapter or section, the title is the book. The title MUST be identified using a unique identifier (e.g. an ISBN for a book) regardless of format (e.g. PDF or HTML).
+For the purpose of COUNTER metrics, a title represents the parent work (book) that a specific content item is part of. The title MUST be identified using a unique identifier (e.g. an ISBN for a book) regardless of format (e.g. PDF, HTML or EPUB).
+
+While the method for counting book usage in R5.1 at the item level is different than it was in R5, the method for counting title-level usage is unchanged. That means Unique_Title_Requests and Unique_Title_Investigations are comparable across report providers.
```

Comment:
Comparable across releases (instead of report providers, since you are comparing releases)?

Reply:
both - the concerns seem to be mostly around cross-provider comparisons